.github/workflows/ci.yml

```diff
@@ -3,10 +3,11 @@ name: ci
 on:
   push:
     branches:
+      - alpha
+      - beta
```

Comment:
suggestion: I don't think that I will ever use alpha / beta. I prefer to use rc.X convention. Maybe you could add a wildcard branch here, something like `**-rc.**`

Reply:
Changed :raised_hands: